aten/src/ATen/core/PythonFallbackKernel.cpp

```diff
@@ -58,8 +45,10 @@ struct StashTLSOnEntryGuard {
 
 void pythonFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
   TORCH_INTERNAL_ASSERT(tls_on_entry.has_value());
-  c10::impl::ForceDispatchKeyGuard dispatcher_guard(tls_on_entry.value());
-  StashTLSOnEntryGuard stash_guard;
+  // c10::impl::ForceDispatchKeyGuard dispatcher_guard(tls_on_entry.value());
+  // StashTLSOnEntryGuard stash_guard;
```

Comment:
@albanD why is this commented out? Can we delete it?

Reply:
Yes it can be removed. The restore that the user has access to is doing this (if they ask for it).